lang/cs/Org.Apache.REEF/Org.Apache.REEF.nuspec

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0"?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<package >
+  <metadata>
+    <id>Org.Apache.REEF</id>
+    <version>$version$</version>
+    <title>Org.Apache.REEF</title>
+    <authors>The Apache REEF project</authors>
+    <owners>The Apache REEF project</owners>
+    <licenseUrl>http://www.apache.org/licenses/LICENSE-2.0</licenseUrl>
+    <projectUrl>http://reef.apache.org/</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Provide single entry point that contains all necessary REEF/WAKE/TANG projects.</description>
+    <copyright>The Apache Software Foundation</copyright>
+    <tags>Single entry point to reference all Reef/Wake/Tang projects </tags>
+    <dependencies>
```

Comment:
Why would all of these be mentioned as dependencies again? I thought the idea was to package the DLLs from these into a single NuGet?